src/main/java/org/opensearch/ad/task/ADTaskManager.java

```diff
@@ -874,7 +874,7 @@ public void stopDetector(
 
                 consumer.accept(detector);
             } catch (Exception e) {
-                String message = "Failed to start anomaly detector " + detectorId;
+                String message = "Failed to get anomaly detector " + detectorId;
```

Comment:
How about change the error message as "Failed to parse anomaly detector "

Reply:
changed